firebase.json

```diff
@@ -10,112 +10,162 @@
       "src",
       "**/node_modules/**"
     ],
+    "rewrites": [],
     "redirects": [
       { "regex": "^/(flutter(/.*)?)", "destination": "https://api.flutter.dev/:1", "type": 301 },
       { "regex": "^/(objcdoc(/.*)?)", "destination": "https://api.flutter.dev/:1", "type": 301 },
       { "regex": "^/(javadoc(/.*)?)", "destination": "https://api.flutter.dev/:1", "type": 301 },
       { "regex": "(?P<basename>.*)\\.html$", "destination": ":basename", "type": 301 },
       { "regex": "(?P<basename>.*)\\.$", "destination": ":basename", "type": 301 },
+
+      { "source": "/community", "destination": "https://flutter.dev/community", "type": 301 },
+      { "source": "/showcase", "destination": "https://flutter.dev/showcase", "type": 301 },
```

Comment:
This doesn't seem right - what will happen when this gets deployed to `flutter.dev`?

Reply:
Nevermind, I forgot that this is being deployed to `docs.flutter.dev` now.